site/docs/hosting/vercel.md

```diff
@@ -70,35 +70,35 @@ It should look roughly like this:
 ```ts
 import { Bot, webhookCallback } from "grammy";
 
-const bot = new Bot(process.env.BOT_TOKEN); // <-- insert your bot token from environment variable
+const bot = new Bot(process.env.BOT_TOKEN);
```

Comment:
I just noticed that this won't typecheck

Reply:
This = the code examples?

Reply:
This line. On Node, the env value may be absent but the Bot constructor requires the token.

Reply:
Do you refer to this?

```ts
if (process.env.BOT_TOKEN === undefined) {
  throw new TypeError('BOT_TOKEN must be provided!');
}

const token: string = process.env.BOT_TOKEN;
const bot = new Bot(token);
```

Reply:
@KnorpelSenf Yes, should we do something like that?^

Reply:
Yes, thanks!

Just check the variable token directly ;)